lib/MusicBrainz/Server/Controller/Track.pm

```diff
@@ -37,6 +37,8 @@ sub show : Chained('load') PathPart('')
     if (defined ($c->stash->{recording}))
     {
         $uri = $c->uri_for_action('/recording/show', [ $c->stash->{recording}->gid ]);
+        # The track link is now a recording link: this should be considered a permanent move
```

Comment:
This is unclear to me, can you please provide an example URL?

Reply:
https://musicbrainz.org/track/56b85b37-d64b-4be6-ae3d-04d730b6dfcf

Reply:
Thanks, I thought it was about some track links, it is just about any track link.